packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -87,6 +87,10 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
     return this.input.current && this.input.current.inputElement;
   }
 
+  public get selectedItems(): T[] {
+    return this.selectedItemsList.current ? this.selectedItemsList.current.selectedItems() : [];
```

Comment:
I strongly advise against this. Instead users should be using onchanged to track it. If we must expose it like this be aware of the potential interaction when some implementations of basepicker autoselect the first item in the suggestions list when it gets resolved.

Reply:
I see, I'll remove this then. Thanks!